{
  "contest": 342,
  "handle": "VAD37",
  "risk": "3",
  "title": "Complication of problems with `V3Vault.repay()`. It will frequently fail on MAINNET. Possible damage to user finance",
  "issueId": 371,
  "issueUrl": "https://github.com/code-423n4/2024-03-revert-lend-findings/issues/371"
}